packages/destination-actions/src/destinations/fullstory/index.ts

```diff
@@ -21,6 +22,12 @@ const destination: DestinationDefinition<Settings> = {
       subscribe: 'type = "identify"',
       partnerAction: 'identifyUser',
       mapping: defaultValues(identifyUser.fields)
+    },
+    {
+      name: 'Identify User V2',
+      subscribe: 'type = "identify"',
+      partnerAction: 'identifyUserV2',
+      mapping: defaultValues(identifyUserV2.fields)
```

Comment:
By including this as a preset, would a new FullStory cloud mode destination automatically get `identifyUserV2` configured? If so, I don't think we want that.

Reply:
I needed to add this to get the option to display in the test app. I don't know how to test your question.